src/Frontend/.eslintrc.cjs

```diff
@@ -1,33 +0,0 @@
-/* eslint-env node */
-require("@rushstack/eslint-patch/modern-module-resolution");
-
-module.exports = {
-  root: true,
-  env: {
-    node: true,
-    es6: true,
-  },
-  ignorePatterns: ["node_modules/**/*", "dist/**/*", "*.ts"],
-  extends: ["plugin:vue/vue3-essential", "eslint:recommended", "plugin:@typescript-eslint/recommended", "@vue/eslint-config-prettier/skip-formatting"],
-  parser: "vue-eslint-parser",
-  plugins: ["@typescript-eslint", "github", "import", "unused-imports", "prefer-arrow"],
-  parserOptions: {
-    ecmaVersion: "latest",
-    parser: "@typescript-eslint/parser",
-    extraFileExtensions: [".vue"],
-  },
-  rules: {
-    "prettier/prettier": "error",
-    "require-await": "error",
-    "no-await-in-loop": "error",
-    "github/no-then": "error",
```

Comment:
couldn't find repacement

Reply:
i would suggest using the deprecated eslintrc until there is a replacement for these

Reply:
We can't, we need to move out of it.
![image](https://github.com/user-attachments/assets/d4f41165-3bfc-4a3b-a3b6-cca8f7b97eaf)

And those rules are not needed, we have moved to async await now.

Reply:
those rules are for ensuring future work follows the async/await paradigm. The blog says that eslintrc will not be maintained, but also won't be removed. I'm not suggesting staying on eslintrc long term, just until all the rules we want to enforce are supported

Reply:
the problem is that those are not going to be updated, so we do need to look for replacements

Reply:
replaced with https://github.com/eslint-community/eslint-plugin-promise/blob/main/docs/rules/prefer-await-to-then.md